deploy/template.yaml

```
@@ -225,6 +225,10 @@ Resources:
               BACKEND_STACK_NAME: demo-sam-app
               FRONTEND_STACK_NAME: node-app
               K6_DYNATRACE_DASHBOARD_ID: 1e845440-d013-4472-9d65-2ea21a5cb41a
+              TEST_CLIENT_EXECUTE_URL: ${TEST_CLIENT_EXECUTE_URL}
+              BACK_END_URL: ${BACK_END_URL}
+              FRONT_END_URL: ${FRONT_END_URL}
+              BIOMETRIC_SESSION_ID: ${BIOMETRIC_SESSION_ID}
             parameter-store:
```

```suggestion
            parameter-store:
              TEST_CLIENT_EXECUTE_URL: "/perfTest/testClientUrl"
              BACK_END_URL: "/perfTest/backendUrl"
              FRONT_END_URL: "/perfTest/frontendUrl"
              BIOMETRIC_SESSION_ID: "/perfTest/biometricSessionID"
```
Would suggest moving these to parameter store